SDVModTest/ModConfig.cs

```diff
@@ -7,73 +7,103 @@
 
 namespace UIInfoSuite
 {
-    class ModConfig
+    public class ModConfig

```

Comment:
What reason is there for making this class public? It shouldn't be accessed from outside this project.

Reply:
in order for the framework to save to and load from the config to a json the config class must be visible to the framework